pom.xml

```diff
@@ -30,7 +41,7 @@
     <dependency>
       <groupId>com.razorpay</groupId>
       <artifactId>razorpay-java</artifactId>
-      <version>1.0.0</version>
+      <version>1.2.0-SNAPSHOT</version>
```

Comment:
if you have tested `1.2.0` then first release that, and use `1.2.0`, not the snapshot version